royalty/princess

```diff
@@ -0,0 +1,6 @@
+Princess- (4)
+- Takes the place of the King, if they are alive and there is no alive Prince or Queen when the King dies.
+- The first time the Princess is about to die to tribute, the Princess will not die, however will be revealed to everyone. This ability is then lost.
+- The Princess will receive all of the money paid towards tribute when the passive ability is activated.
```

Comment:
should clarify, the combined money paid by all teams

Reply:
I think this is resolved